scripts/variantstore/gvs-overview.md

```diff
@@ -0,0 +1,165 @@
+
+The GVS beta workspace in Terra is read-only, so you’ll need to clone the workspace to create a copy where you can upload your own data and run the workflow. For instructions on how to clone workspaces in Terra, see the article [Make your own project workspace](https://support.terra.bio/hc/en-us/articles/360026130851). For a tour of workspaces in Terra, see [Working with workspaces](https://support.terra.bio/hc/en-us/articles/360024743371).
+
+### Workflow requirements
+
+#### Set up BigQuery
+
+To start using the GVS workflow in Terra, you need to create a BigQuery dataset with permissions that will allow Terra to access the dataset by following the instructions below:
+
+**1.** First, **create a dataset in BigQuery** by following the instructions in [Creating datasets](https://cloud.google.com/bigquery/docs/datasets) from the BigQuery documentation.
+
+**2.** Find your Terra proxy group by **selecting the main menu** (three horizontal line icon) at the top left of any Terra page. Next, **click on your account name**, followed by **Profile**. Here, you should see a field called **Proxy Group**, which lists your proxy group underneath. For more information on proxy groups in Terra, see [Pet service accounts and proxy groups](https://support.terra.bio/hc/en-us/articles/360031023592). 
+
+**3.** Grant your proxy group the **BigQuery Data Editor** role on the dataset you created in Step 1 by following the instructions in [Controlling access to datasets](https://cloud.google.com/bigquery/docs/dataset-access-controls) from the BigQuery documentation.
+
+**4.** Grant your proxy group the **BigQuery Data Editor**, **BigQuery Job User**, and **BigQuery Read Session User** roles on the Google Project that contains the dataset you created in Step 1 by following the instructions in [Manage access to projects, folders, and organizations](https://cloud.google.com/iam/docs/granting-changing-revoking-access) from the Google Cloud documentation.
+
+#### Reblocked gVCF files
+
+The GVS workflow takes in reblocked gVCF files as input. If your files are not already reblocked, you can reblock them using the [WARP reblocking workflow](https://github.com/broadinstitute/warp/blob/master/pipelines/broad/dna_seq/germline/joint_genotyping/reblocking/ReblockGVCF.wdl), which is configured in the [ReblockGVCF Terra workspace](https://app.terra.bio/#workspaces/warp-pipelines/ReblockGVCF). For more information about reblocking, check out [WARP Whole Genome and Exome Pipelines Produce Reblocked GVCFs](https://broadinstitute.github.io/warp/blog/tags/reblock/).
+
+#### gVCF annotations
+
+Input gVCF files for the GVS workflow must include the annotations described in the table below:
```

Comment:
Maybe it would be helpful to call out earlier that the gVCFs must have been created with a GATK "functional equivalence" compatible solution (basically, HaplotypeCaller, DRAGEN, Sention).  That's where these annotations come from and might make this easier to understand.    The methods folks might have good links for this